examples/oft-aptos/README.md

```diff
@@ -0,0 +1,135 @@
+```
+
+Make sure to set the account that will be used for administrative functions such as settting the peers should be set in the move.toml as OFT_ADMIN
+
+Account Descriptions:
+- oft is the owner of the deployer object which gives the account that owns the object the abilitiy to upgrade the object and also burn and mint the fungible asset.
+- oft_admin is the account that is responsible for administrative functions such as setting the peers, set uln config... etc.
+
+The address of the deployed oft is printed out in the terminal:
+```
+Code was successfully deployed to object address 0xaebb730cc67b4b0987ec99cd20b9eaf7c5c0d517f4424ab4f4b1450c6c8d9bb4.
+{
+  "Result": "Success"
+}
+```
+## Set the delegate:
+Run the following command to set the delegate to the oft. Ensure first that you have specified the delegate address in the layerzero.config.ts file.
+```
+npx hardhat run tasks/move/setDelegate.ts
+```
+## Initialize the OFT:
+Inside the file tasks/move/initAptosOFT.ts, set the following parameters:
+```typescript
+const tokenName = '<your-token-name>'
+const tokenSymbol = '<your-token-symbol>'
+const iconUri = '<your-icon-uri>'
+const projectUri = '<your-project-uri>'
+const sharedDecimals = <your-shared-decimals>
+const localDecimals = <your-local-decimals>
+```
+Then run the following command to initialize the oft:
+```
+ts-node tasks/move/initAptosOFT.ts
+```
+## Wiring your aptos OFT:
+
+#### layerzero.config.ts:
+ensure that all of the contracts you wish to wire your aptos oft to are included under the contracts array and connections array. For example, if you want to wire your aptos oft to fuji, you would include both contracts under the contracts array, and specify them like this:
+```typescript
+const aptosContract: OmniPointHardhat = {
+    eid: EndpointId.APTOS_TESTNET,
+    contractName: 'oft',
+}
+
+const fujiContract: OmniPointHardhat = {
+    eid: EndpointId.AVALANCHE_V2_TESTNET,
+    contractName: 'MyOFT',
+}
+
+const config: OAppOmniGraphHardhat = {
+    contracts: [
+        {
+            contract: fujiContract,
+        },
+        {
+            contract: aptosContract,
+            config: {
+                delegate: '<your-aptos-account-address>',
+                owner: '<your-aptos-owner-address>',
+            },
+        },
+    ],
+    connections: [
+        {
+            from: aptosContract,
+            to: fujiContract,
+        },
+        {
+            from: aptosContract,
+            to: sepoliaContract,
+        },
+    ],
+}
+```
+
+IMPORTANT: Before running the wire script ensure that you have run aptos init and configuered your desired network such as custom, testnet, mainnet, etc. and have also deployed the oft to that network.
+```
+npx hardhat run tasks/move/wireAptosOFT.ts
+```
+
+## Scripts For internal testing:
+These scripts will not be shipped in the final version of the aptos cli. They are only for internal testing purposes.
```

Comment:
If these scripts aren't supposed to be shipped in the final version of the aptos-cli branch then why they are part of this PR? Should we remove it?

Reply:
this pr was for feedback on the dev ig. its still under development